awx_collection/plugins/modules/tower_workflow_launch.py

```diff
@@ -36,25 +59,21 @@
       required: False
       default: True
       type: bool
+    interval:
+      description:
+        - The interval to request an update from Tower.
+      required: False
+      default: 1
+      type: float
     timeout:
       description:
         - If waiting for the workflow to complete this will abort after this
           amount of seconds
       type: int
-
-requirements:
-  - "python >= 2.6"
-  - ansible-tower-cli >= 3.0.2
-
 extends_documentation_fragment: awx.awx.auth
 '''
 
 RETURN = '''
-tower_version:
-    description: The version of Tower we connected to
-    returned: If connection to Tower works
-    type: str
-    sample: '3.4.0'
 job_info:
     description: dictionary containing information about the workflow executed
```

Comment:
It's just a dictionary with a single id in it, so I don't think this description is accurate anymore. If this conversion triggers a backward-incompatible change, I think that's fine.